dist/redhat/scylla-machine-image.spec

```diff
@@ -36,6 +36,7 @@ install -d -m755 $RPM_BUILD_ROOT/opt/scylladb/scylla-machine-image
 install -d -m755 $RPM_BUILD_ROOT/opt/scylladb/scylla-machine-image/lib
 install -m644 lib/log.py lib/scylla_cloud.py lib/user_data.py $RPM_BUILD_ROOT/opt/scylladb/scylla-machine-image/lib
 install -m644 common/aws_io_params.yaml $RPM_BUILD_ROOT/opt/scylladb/scylla-machine-image/
+install -m644 common/aws_net_params.json $RPM_BUILD_ROOT/opt/scylladb/scylla-machine-image/
```

Comment:
I don't fully understand why we need yet another YAML, only this time it is a JSON file, which has a different set of EC2 instance types. Now when we wish to edit/add instance, we need to different files edited, in 2 different formats.
Just add a single parameter to aws_io_params.yaml - base network throughput.

Reply:
The file in question can be auto generated by a simple call to aws cli (which I forgot to include a comment about in this PR). It can only output json. The point of using the generated file, and not either manually create it or create a formatting routine is that it remains trivial to generate using existing, non-scylla tools.

Reply:
Maybe when we'll have enough outputs from various tools, like the one in question and maybe similar for GCP, Azure, etc. Maybe when these pre-created configuration files are piled up - we'll add yet another script to combine them together into a one coherent source of truth.
I don't see the urgency now.